testing/scripts/create-qe-tier1-tickets.py

```
@@ -0,0 +1,142 @@
+                "project": {"key": "MTV"},
+                "summary": ticket["summary"],
+                "issuetype": {"name": "Story"},
+                "components": [{"id": COMPONENT_ID}],
+                "labels": ["user-interface"],
+                "fixVersions": [{"id": FIX_VERSION_ID}],
+                "customfield_10020": SPRINT_ID,
+                "customfield_10028": float(ticket["story_points"]),
+                "description": ticket["description"],
+            }
+        }
+        response = requests.post(f"{JIRA_BASE}/rest/api/2/issue", headers=HEADERS, json=payload)
```

Ruff (0.15.17)

[error] Probable use of `requests` call without timeout

(S113)